examples/UpdateContractMint/Program.cs

```diff
@@ -0,0 +1,260 @@
+                    var json = evt.GetDeserializeEvent(moduleSchema, receiveName.GetContractName());
+                    Console.WriteLine($"- {json}");
+                }
+            }
+            else
+            {
+                // For other contracts we just log the event bytes as a hex encoded string.
+                foreach (var evt in update.Item2)
+                {
+                    Console.WriteLine($"- {evt.ToHexString()}");
+                }
+            }
+        }
+    }
+
+    /// <summary>
+    /// Class for constructing the JSON representation of the smart contract parameter for 'mint'
+    /// entrypoint of 'cis2_multi' contract.
+    /// The exact structure will depend on the details of the smart contract module.
+    /// </summary>
+    private class MintParameter
+    {
+        /// <summary>
+        /// The owner for the newly minted tokens.
+        ///
+        /// This property only represents the owner being an account address, but the smart contract
+        /// also allow for a contract address here, which we choose not include in the type for
+        /// simplicity.
+        /// </summary>
+        [JsonPropertyName("owner")]
+        public JsonAccountAddr Owner { get; set; }
+        /// <summary>
+        /// The metadata url for a newly minted token type.
+        /// </summary>
+        [JsonPropertyName("metadata_url")]
+        public JsonMetadataUrl MetadataUrl { get; set; }
+        /// <summary>
+        /// The TokenID identifying the token type to mint.
+        /// </summary>
+        [JsonPropertyName("token_id")]
+        public string TokenId { get; set; }
+    }
+
+    /// <summary>
+    /// Indicator class for some address being an account address.
+    /// Note this only represents a single account address, but still contains an array of strings
+    /// for the JSON format to match.
+    /// </summary>
+    private class JsonAccountAddr
+    {
+        public string[] Account { get; set; }
+    }
+
+    /// <summary>
+    /// Class for constructing the JSON representation of the token metadata URL.
+    /// </summary>
+    private class JsonMetadataUrl
+    {
+        /// <summary>
+        /// The URL pointing to the token metadata.
+        /// </summary>
+        [JsonPropertyName("url")]
+        public string Url { get; set; }
+        /// <summary>
+        /// The hash of the token metadata.
+        ///
+        /// This property only supports not providing a content hash of the metadata, but the smart
+        /// contract supports this as well, we choose not include this in the type for simplicity.
+        /// </summary>
+        [JsonPropertyName("hash")]
+        public JsonNoHash ContentHash { get; set; }
+    }
+
+    /// <summary>
+    /// Indicator of no hash provided for the token metadata.
+    /// </summary>
+    private class JsonNoHash
+    {
+        public string[] None { get; set; }
```

Comment:
Why is this a string array?

Reply:
The JSON needs to be `{ "None": [] }`, so any array will do.
I have extended on this in the comment